examples/iOS/CMakeLists.txt

```diff
@@ -1,16 +1,16 @@
 
-set(SRCROOT ${PROJECT_SOURCE_DIR}/examples/ios)
+set(SRCROOT ${PROJECT_SOURCE_DIR}/examples/iOS)
 set(CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/cmake/Modules)
 
 # All source files
 set(SRC ${SRCROOT}/main.cpp)
 
 # Use the resources from the android example
 set(RESOURCES
-    ${CMAKE_CURRENT_SOURCE_DIR}/../android/assets/canary.wav
-    ${CMAKE_CURRENT_SOURCE_DIR}/../android/assets/image.png
-    ${CMAKE_CURRENT_SOURCE_DIR}/../android/assets/orchestral.ogg
-    ${CMAKE_CURRENT_SOURCE_DIR}/../android/assets/sansation.ttf)
+    ${PROJECT_SOURCE_DIR}/examples/android/assets/canary.wav
```

Comment:
I might have expressed myself a bit unclear in my last comment. I think it isn't too good of an idea to cross-reference resources from another example. To be completely clean, the iOS example should have a resources subdirectory with all its required resources like all the other examples. I would simply just copy over the files from the android assets directory into a new resources subdirectory so the final directory structure matches that of the other examples.